.github/workflows/tox.yml

```diff
@@ -211,6 +211,15 @@ jobs:
         python3 -m tox
       env:
         TOXENV: ${{ matrix.tox_env }}-devel
+    - name: Combine coverage data
+      # produce a single .coverage file at repo root
+      run: coverage combine .tox/.coverage.*
+    - name: Upload coverage data
+      uses: codecov/codecov-action@v1
```

Comment:
Note: we need to use v2 because v1 is deprecated (not sure for how long it'll be around).